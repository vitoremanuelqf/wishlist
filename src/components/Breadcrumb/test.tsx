import { render, screen } from '@testing-library/react'

import { Breadcrumb } from '.'

describe('<Breadcrumb />', () => {
  it('should render the correct links', () => {
    const { container } = render(
      <Breadcrumb
        routes={[
          { name: 'Home', url: '/' },
          { name: 'Wishlist', url: '/wishlist' },
        ]}
      />,
    )

    expect(screen.getByText(/Home/i)).toBeInTheDocument()
    expect(screen.getByText(/Wishlist/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
