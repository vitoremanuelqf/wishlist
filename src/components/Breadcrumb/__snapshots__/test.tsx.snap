// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> should render the correct links 1`] = `
<div
  class="flex h-auto w-full items-center border-b-[1px] border-b-gray-300 py-2 sm:py-4"
>
  <a
    class="mr-1 text-xl text-[#5b2b84] sm:text-2xl font-normal"
    href="/"
  >
    Home
     /
  </a>
  <a
    class="mr-1 text-xl text-[#5b2b84] sm:text-2xl font-bold"
    href="/wishlist"
  >
    Wishlist
  </a>
</div>
`;
